<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>1980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务端</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>选择客户端</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,0,10">
        <item>
         <widget class="QLineEdit" name="lineEdit_sp">
          <property name="placeholderText">
           <string>请输入待监听端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectToClient">
          <property name="text">
           <string>监听端口</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEdit_ss">
          <property name="placeholderText">
           <string>请输入待发送数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s_send">
          <property name="text">
           <string>发送目标数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_sa">
          <property name="placeholderText">
           <string>接受数据如下</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>输出：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_console"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>控制</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_readKeys">
          <property name="text">
           <string>读取按键状态</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_key1">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>按钮1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_key2">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>按钮2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_key3">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>按钮3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_directControl">
          <property name="text">
           <string>直接控制灯</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_led1">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>灯1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_led2">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>灯2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_led3">
          <property name="styleSheet">
           <string notr="true">border-style:hidden;</string>
          </property>
          <property name="text">
           <string>灯3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_recLed">
          <property name="text">
           <string>记录灯动作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_recOperate">
          <property name="text">
           <string>记录操作记录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
